libsolutil/StringUtils.h

```diff
@@ -197,4 +199,45 @@ inline std::optional<unsigned> toUnsignedInt(std::string const& _value)
 	}
 }
 
+/// Converts parameter _c to its lowercase equivalent if c is an uppercase letter and has a lowercase equivalent. It uses the classic "C" locale semantics.
+/// @param _c value to be converted
+/// @return the converted value
+inline char toLower(char _c)
+{
+	return tolower(_c, std::locale::classic());
+}
+
+/// Converts parameter _c to its uppercase equivalent if c is an lowercase letter and has a uppercase equivalent. It uses the classic "C" locale semantics.
+/// @param _c value to be converted
+/// @return the converted value
+inline char toUpper(char _c)
+{
+	return toupper(_c, std::locale::classic());
+}
+
+/// Converts parameter _s to its lowercase equivalent. It uses the classic "C" locale semantics.
+/// @param _s value to be converted
+inline void toLower(std::string& _s)
```

Comment:
Can you change this so that it returns a copy of the string? This way it might be rather error prone.

Reply:
Sure! Fixup commit already added to the PR.